pkg/kubelet/apis/kubeletconfig/validation/validation.go

```diff
@@ -31,6 +31,11 @@ import (
 func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error {
 	allErrors := []error{}
 
+	// Make a local copy of the global feature gates and combine it with the gates set by this configuration.
+	// This allows us to validate the config against the set of gates it will actually run against.
+	localFeatureGate := utilfeature.DefaultFeatureGate.DeepCopy()
```

Comment:
why are we starting with a copy of the global gates? wouldn't kc.FeatureGates be the merged args+config at this point?

Reply:
yup, exactly